google/cloud/security/common/gcp_type/instance.py

```diff
@@ -55,13 +54,21 @@ def __init__(self, **kwargs):
 
     @property
     def key(self):
-        """Returns a Key identifying the object.
+        """Return a Key identifying the object.
 
         Returns:
             Key: the key
         """
         return Key.from_args(self.project_id, self.zone, self.name)
 
+    def create_network_interfaces(self):
+        """Return a list of network_interface objects.
+
+        Returns:
+            List: list of InstanceNetworkInterface objects
+        """
+        return [InstanceNetworkInterface(**ni) for ni in self.network_interfaces]
```

Comment:
I think I've managed to split iterators something like:

```
[x
 for x in some_list]
```

does that work for you?